ons_alpha/urls.py

```diff
@@ -15,21 +13,27 @@
 from ons_alpha.utils.cache import get_default_cache_control_decorator
 
 
-private_urlpatterns = []
+# Private URLs are not meant to be cached.
+private_urlpatterns = [
+    path("documents/", include(wagtaildocs_urls)),
+]
+
+if apps.is_installed("wagtail.admin"):
+    from wagtail.admin import urls as wagtailadmin_urls
+
+    private_urlpatterns.append(path("admin/", include(wagtailadmin_urls)))
```

Comment:
Thought: Should the writable instance be _only_ the admin, and not allow serving the regular site? Perhaps just redirecting to the admin?

Reply:
Not serving the regular would make local development a pain, as we would have to run both and update the routing to use the read-only app. I don't see an added risk of allowing the regular site via admin instance. I want to reduce the no. of moving parts/variants in delivering this solution.